database/buildSchema/43_views_functions_triggers.sql

```diff
@@ -1750,12 +1750,12 @@ LANGUAGE plpgsql;
 
 
 -- Trigger for the above on review_assignment
-DROP TRIGGER IF EXISTS update_application_reviewer_stats ON public.review_assignment;
-CREATE TRIGGER update_application_reviewer_stats
-    AFTER INSERT OR UPDATE OF status, allowed_sections, assigned_sections
-     ON public.review_assignment
-    FOR EACH ROW
-    EXECUTE FUNCTION public.update_application_reviewer_stats ();
+-- DROP TRIGGER IF EXISTS update_application_reviewer_stats ON public.review_assignment;
+-- CREATE TRIGGER update_application_reviewer_stats
+--     AFTER INSERT OR UPDATE OF status, allowed_sections, assigned_sections
+--      ON public.review_assignment
+--     FOR EACH ROW
+--     EXECUTE FUNCTION public.update_application_reviewer_stats ();
```

Comment:
> since it affects a lot of files but does nothing functional

Okay, not 100% correct, I did make a start on this before re-branching. But don't worry about this, it's removed completely in the main PR.